tests/validation/tests/v3_api/test_run_sonobuoy.py

```diff
@@ -0,0 +1,50 @@
+import base64
+import os
+import time
+from .common import *
+
+RANCHER_SONOBUOY_VERSION = os.environ.get("RANCHER_SONOBUOY_VERSION", "0.18.2")
+RANCHER_K8S_VERSION = os.environ.get("RANCHER_K8S_VERSION", "v1.18.2")
+RANCHER_SONOBUOY_MODE = os.environ.get("RANCHER_SONOBUOY_MODE",
+                                       "certified-conformance")
+RANCHER_KUBECONFIG = os.environ.get("RANCHER_KUBECONFIG")
+
+
+def test_sonobuoy_results():
+    config = base64.b64decode(RANCHER_KUBECONFIG).decode("utf-8")
+    kubeconfig = "./kubeconfig"
+
+    with open(kubeconfig, 'w') as f:
+        f.write(config)
+
+    cmd = "sonobuoy run --mode={0} --kube-conformance-image-version={1} --kubeconfig={2}".format(RANCHER_SONOBUOY_MODE, RANCHER_K8S_VERSION, kubeconfig)
+    status = run_command(cmd)
+    time.sleep(60)
+
+    cmd = "sonobuoy status --kubeconfig={0}".format(kubeconfig)
+    status = run_command(cmd)
+    print(status)
+    while "running" in status or "Pending" in status:
```

Comment:
Could we get this in a separate method in this file? like `wait_for_sonobouy_results`